hadoop-dataset-extractor-standalone/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
   extraLibs externalDependency.avro_mapred
   extraLibs externalDependency.hive_exec
   extraLibs externalDependency.pig
+  extraLibs 'org.apache.parquet:parquet-avro:1.8.1'
```

Comment:
Please move this to externalDependency here: https://github.com/linkedin/WhereHows/blob/master/build.gradle#L85